js/project-list.js

```diff
@@ -0,0 +1,47 @@
+/* Project List */
+
+var ProjectList = {
+  template: _.template($("script#project-list-template").html()),
+  buildHTML: function(projectData) {
+    var id = projectData.id;
+    var timestamp = new Date(projectData.Timestamp);
+    var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][timestamp.getMonth()];
+    var issues = projectData.Issues ? projectData.Issues : "";
+    var issuesAsArray = issues.split(", ");
+    var dataForTemplate = {
+      id: id,
+      rawTimestamp: Date.parse(projectData.Timestamp),
+      featured: projectData.Featured,
+      starred: FavouritesManager.isProjectFavourited(id),
+      title: projectData.Title ? projectData.Title : "",
+      creators: projectData.Creators ? projectData.Creators : "",
+      timestamp: month + " " + timestamp.getDate() + ", " + timestamp.getFullYear(),
+      interest: projectData.Interest ? projectData.Interest : "",
+      networkConnection: projectData['Network connection'] ? projectData['Network connection'] : "",
+      origin: projectData.Origin ? projectData.Origin : "",
+      detailViewLink: utility.getProjectDirectLink(id),
+    };
+    return this.template(dataForTemplate);
+  },
+  readMoreLinkClickHandler: function(event) {
```

Comment:
We are missing proper event handler binding. Therefore this click handler is not being invoked at all. We should attach it to the `title` element of project list item in DOM.